ui/src/styles.css

```diff
@@ -353,6 +353,21 @@ form .form-block > label,
   color: #262262 !important
 }
 
+.p-accordion-header-link {
```

Comment:
These are all overrides to PrimeReact's accordion theming.

Later on, we can either make our own them or replace this with something outside of PrimeReact.